moderation-commands/clean.md

```diff
@@ -25,18 +25,25 @@ For more information on editing permission requirements for specific users/roles
 
 ### Options
 
-* `amount`: <TODO: describe>
-* `user`: <TODO: describe>
-* `starts-with`: <TODO: describe>
-* `ends-with`: <TODO: describe>
-* `contains`: <TODO: describe>
-* `bots`: <TODO: describe>
-* `system`: <TODO: describe>
-* `preserve-pins`: <TODO: describe>
-* `no-trace`: <TODO: describe>
+* `amount`: The amount of messages you want to clean (The default is 100)
+* `user`: The user you want to do delete the messages of
+* `starts-with`: Delete messages that start with this value
+* `ends-with`: Delete messages that end with this value
+* `contains`: Delete messages that contain this value
+* `bots`: Whether or not to delete messages sent by bots (The default is false)
```

Comment:
this works a bit differently whether it's explicitly false or "not set", refer to https://canary.discord.com/channels/359057740200673280/902924561073446953/1254732734169874523